journey-map/public/index.mjs

```diff
@@ -93,12 +101,15 @@ const getPageUrl = (id, context) => {
     return context ? `${baseUrl}?context=${encodeURIComponent(context)}` : baseUrl;
 }
 
-const getJourneyUrl = (id) => `?journeyType=${encodeURIComponent(id)}`;
+const getJourneyUrl = (id) => `?${JOURNEY_TYPE_SEARCH_PARAM}=${encodeURIComponent(id)}`;
+
+const getNestedJourneyUrl = (id) => `?${NESTED_JOURNEY_TYPES}=${encodeURIComponent(id)}`;
 
 const switchJourney = async (targetJourney, targetState) => {
     // Update URL
     const params = new URLSearchParams(window.location.search);
-    params.set('journeyType', targetJourney);
+    params.delete(NESTED_JOURNEY_TYPE_SEARCH_PARAM);
```

Comment:
when switching to a different journey, we want to remove the `nestedJourneyType` so that it doesn't just show the nested journey map